tests/test_flows.py

```diff
@@ -366,10 +365,12 @@ def test_ap_capability_query(self):
 
         debug("Confirming ap capability query has been received on agent")
         self.check_log(env.agents[0], "AP_CAPABILITY_QUERY_MESSAGE")
-
+        capability_query = env.wired_sniffer.get_packets_for_cmdu_type(self.running,0x8001)
+        
         debug("Confirming ap capability report has been received on controller")
         self.check_log(env.controller, "AP_CAPABILITY_REPORT_MESSAGE")
-
+        capability_report = env.wired_sniffer.get_packets_for_cmdu_type(self.running,0x8002)
```

Comment:
Please remember to run `flake8` on your python scripts and fixing issues it notices. For example, here it would complain about not having a space after the comma.